inst/templates/ad_advs.R

```diff
@@ -167,18 +168,16 @@ param_lookup <- tibble::tribble(
   "HEIGHT", "HEIGHT", "Height (cm)", 2,
   "BMI", "BMI", "Body Mass Index(kg/m^2)", 3,
   "HDCIRC", "HDCIRC", "Head Circumference (cm)", 4,
-  NA_character_, "WTASDS", "Weight-for-age z-score", 5,
-  NA_character_, "WTAPCTL", "Weight-for-age percentile", 6,
-  NA_character_, "BMISDS", "BMI-for-age z-score", 7,
-  NA_character_, "BMIPCTL", "BMI-for-age percentile", 8,
-  NA_character_, "HDCSDS", "HDC-for-age z-score", 9,
-  NA_character_, "HDCPCTL", "HDC-for-age percentile", 10,
-  NA_character_, "WGTHSDS", "Weight-for-length/height Z-Score", 11,
-  NA_character_, "WGTHPCTL", "Weight-for-length/height Percentile", 12,
-  NA_character_, "BMIHSDS", "BMI-for-length/height Z-Score", 13,
-  NA_character_, "BMIHPCTL", "BMI-for-length/height Percentile", 14,
-  NA_character_, "HDCHSDS", "HDC-for-length/height Z-Score", 15,
-  NA_character_, "HDCHPCTL", "HDC-for-length/height Percentile", 16
+  NA_character_, "WGTASDS", "Weight-for-age z-score", 5,
+  NA_character_, "WGTAPCTL", "Weight-for-age percentile", 6,
+  NA_character_, "HGTSDS", "Height-for-age z-score", 7,
+  NA_character_, "HGTPCTL", "Height-for-age percentile", 8,
+  NA_character_, "BMISDS", "BMI-for-age z-score", 9,
+  NA_character_, "BMIPCTL", "BMI-for-age percentile", 10,
+  NA_character_, "HDCSDS", "HDC-for-age z-score", 11,
+  NA_character_, "HDCPCTL", "HDC-for-age percentile", 12,
```

Comment:
can we have the PARAM for these as "Head Circumference-for-age..." instead of "HDC-for-age" which would not be as meaningful to for example an FDA reviewer.

Reply:
implemented